packages/hop-node/src/watchers/SyncWatcher.ts

```diff
@@ -857,21 +856,18 @@ class SyncWatcher extends BaseWatcher {
     if (!destinationChainId) {
       return
     }
-    const destinationBridge = this.getSiblingWatcherByChainId(destinationChainId).bridge // eslint-disable-line @typescript-eslint/no-non-null-assertion
-    const _transferIds = await destinationBridge.getTransferIdsFromSettleEventTransaction(multipleWithdrawalsSettledTxHash)
+    const destinationBridge = this.getSiblingWatcherByChainId(destinationChainId).bridge
+    const { transferIds: _transferIds, bonder }= await destinationBridge.getParamsFromSettleEventTransaction(multipleWithdrawalsSettledTxHash)
     const tree = new MerkleTree(_transferIds)
     const computedTransferRootHash = tree.getHexRoot()
     if (computedTransferRootHash !== transferRootHash) {
       logger.warn(
         `populateTransferRootTimestamp computed transfer root hash doesn't match. Expected ${transferRootHash}, got ${computedTransferRootHash}. isNotFound: true, List: ${JSON.stringify(_transferIds)}`
       )
       await this.db.transferRoots.update(transferRootHash, { isNotFound: true })
-    } else {
-      await this.db.transferRoots.update(transferRootHash, {
-        transferIds: _transferIds
```

Comment:
is it expected that the transferRoot item should always have the list of transferIds at this point if we're removing the update here?

Reply:
Great catch. For some reason I thought that the db `transferIds` was compared to the calldata `_transferIds`, so they would equal here. I see that is a bad assumption. Fixed with 442503ad1eb91248e130323f1b4ded9df32790c5